### ISSUE TITLE
JAR library built with Gradle has two copies of each class

### ISSUE THREAD

**Issue by ctrueden** [dhale/jtk#16]
When building `edu_mines_jtk.jar` from source using gradle, the resultant library contains two copies of each class. The library built via the old Ant build system does not suffer from this issue:
```
$ ls -l build/*/*.jar
-rw-r--r--  1 curtis  staff  2011420 Nov 25 21:45 build/jar/edu_mines_jtk.jar
-rw-r--r--  1 curtis  staff  4020732 Nov 25 21:46 build/libs/edu_mines_jtk.jar
$ jar tf build/jar/edu_mines_jtk.jar | sort | uniq -c | tail -n 5
   1 edu/mines/jtk/util/UnitsParser$UnitsParserTokenManager.class
   1 edu/mines/jtk/util/UnitsParser.class
   1 edu/mines/jtk/util/UnitsSpecs.class
   1 edu/mines/jtk/util/UnitsTest.class
   1 edu/mines/jtk/util/XmlUtil.class
$ jar tf build/libs/edu_mines_jtk.jar | sort | uniq -c | tail -n 5
   2 edu/mines/jtk/util/UnitsParser$UnitsParserTokenManager.class
   2 edu/mines/jtk/util/UnitsParser.class
   2 edu/mines/jtk/util/UnitsSpecs.class
   2 edu/mines/jtk/util/UnitsTest.class
   2 edu/mines/jtk/util/XmlUtil.class
```
I tried to research a little why this might be the case, but I am new to Gradle and the answer was not obvious to me from the `build.gradle` configuration. I tried commenting out the test and demo config in case that somehow caused this behavior, but it had no effect on the final artifact. So any ideas on how to debug the issue further are welcome.

And just in case it matters:
```
$ gradle -version

------------------------------------------------------------
Gradle 2.8
------------------------------------------------------------

Build time:   2015-10-20 03:46:36 UTC
Build number: none
Revision:     b463d7980c40d44c4657dc80025275b84a29e31f

Groovy:       2.4.4
Ant:          Apache Ant(TM) version 1.9.3 compiled on December 23 2013
JVM:          1.7.0_80 (Oracle Corporation 24.80-b11)
OS:           Mac OS X 10.11.1 x86_64
```

**Comment by dhale** [comment]
Nice catch! I have modified build.grade to fix this problem. The fix seems to work with Gradle 2.9. Would you please check that it works for you with Gradle 2.8?

**Comment by ctrueden** [comment]
Yes, the fix also works for Gradle 2.8. Thank you very much!
